### PR TITLE
Use the translator to find relative paths for tagged pages

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #25387

@cticenhour that was the right solution all along

see https://mooseframework.inl.gov/docs/PRs/25636/site/modules/electromagnetics/filter/index.html

### CODE REVIEW

**Review by cticenhour — COMMENTED** [review]
Thanks for taking another look at this based on the previous review. A couple suggestions on user-facing messages and code comments.

File: python/MooseDocs/extensions/tagging.py [hunk 1]
```diff
@@ -102,15 +102,36 @@ def postExecute(self):
...
-                        link_value = '/' + path_value.replace('.md', '.html')
+                            path_value_cut = path_value.split('content/')[1]
+                        else:
+                            path_value_cut = path_value.split('/')[-1]
+
+                        # Find the relative path from the filter page (assumed at filter/index.html)
+                        # Check that there is no ambiguity
+                        target_page = self.translator.findPages(path_value_cut)
+                        filter_page = self.translator.findPages("filter/index.html")
+                        if len(target_page) != 1:
```
Comment (loganharbour) [inline python/MooseDocs/extensions/tagging.py:116]: why is this a warning?

**Review by loganharbour — DISMISSED** [review]

File: python/MooseDocs/extensions/tagging.py (hunk 1 above)
Comment (GiudGiud) [reply python/MooseDocs/extensions/tagging.py:116]: because it will inevitably happen. Cutting at content/ can yield conflicts with same-named pages

Comment (GiudGiud) [reply python/MooseDocs/extensions/tagging.py:116]: the page will still be indexed though, there just wont be a link

Comment (GiudGiud) [reply python/MooseDocs/extensions/tagging.py:116]: made them errors. People will have to de-conflict their pages if they want to use our shiny new extension

Comment (loganharbour) [reply python/MooseDocs/extensions/tagging.py:116]: I plan on finishing up the explicit fixes in my other PR for moosedocs now that the FY is done. I just verified that it'll work here too.

Comment (cticenhour) [reply python/MooseDocs/extensions/tagging.py:116]: I think warning to error is a good change for this particular check - it does keep it consistent with similar errors in when content is initially retrieved. Thanks @loganharbour

**Review by cticenhour — APPROVED** [review]

**Pull request merged**